src/nunit.v2.driver.tests/NUnit2FrameworkDriverTests.cs

```diff
@@ -55,9 +55,13 @@ public void CreateDriver()
         }
 
         [Test]
-        public void Explore()
+        public void ExploreWithFilter()
```

Comment:
I would keep both tests. Of course __we__ know that only one is needed right now, but it provides protection in case somebody makes a change with a special case for the empty filter.

I think I would also like to see some other tests...

1. A test showing that the resulting test case count within each suite is correct after pruning. Right now you are checking that the count is __unchanged__ as a result of the pruning because `PerformBasicResultChecks` uses a constant for the test case count. I guess you need to pass it an argument.

2. A test that `Runner.Test` is unchanged after pruning. See my comment on  not copying the test tree.